src/Polytopes/PolyhedralFan.jl

```diff
@@ -0,0 +1,246 @@
+###############################################################################
+###############################################################################
+### Definition and constructors
+###############################################################################
+###############################################################################
+
+@doc Markdown.doc"""
+   PolyhedralFan(Rays, Cones)
+
+    A polyhedral fan formed from the Rays by taking the cones corresponding to
+    the indices of the sets in Cones.
+"""
+struct PolyhedralFan
+   pm_fan::Polymake.BigObjectAllocated
+end
+
+"""
+   pm_fan(PF::PolyhedralFan)
+
+Get the underlying polymake BigObject.
+"""
+pm_fan(PF::PolyhedralFan) = PF.pm_fan
+
+
+function PolyhedralFan(Cones::Array{Cone,1})
```

Comment:
ideally this could be changed to take any iterator returning Cone objects; whether it is an `Array{Cone,1}` (= `Vector{Cone}` or ` Set{Cone}` or ... shouldn't matter. Maybe something like this:
```suggestion
function PolyhedralFan(itr)
    Cones = collect(Cone, itr)
```
(BTW, by Julia conventions, I would call it `cones` not `Cones`)

Reply:
So this works out of the box in 1.4 and 1.5, but fails for 1.6 and nightly. Do you have an idea?